utilities/rpp-unittests/HOST_NEW/BatchPD_host_pln3.cpp

```diff
@@ -3666,6 +3691,90 @@ int main(int argc, char **argv)
     {
         // test_case_name = "color_jitter";
         missingFuncFlag = 1;
+        break;
+    }
+    case 82:
+    {
+        test_case_name = "ricap";
+        unsigned int initial_permute_array[images];
+        Rpp32u permutedArray1[images] ;
+        Rpp32u permutedArray2[images] ;
+        Rpp32u permutedArray3[images] ;
+        Rpp32u permutedArray4[images] ;
+
+        Rpp32u cropRegion1[images] ;
+        Rpp32u cropRegion2[images] ;
+        Rpp32u cropRegion3[images] ;
+        Rpp32u cropRegion4[images] ;
+
+         
+        for (uint i = 0; i < images; i++ )
+        {
+            initial_permute_array[i] = i;
+        }
+        randomize(initial_permute_array, images);
+        for (uint i=0;i<images;i++)
```

Comment:
Please use memcpy instead of this for loop. Assuming need to just copy a buffer from initialPermuteArray to permutedArray1. Same comment for next 3 for loops.